### PR TITLE
72 helpers

### PR DESCRIPTION
**Pull request by vorg**

### CODE REVIEW

**Comment by vorg** [comment]
@simonharrisco Note about comment convention: please use imperative style

bbox ->  Finish bbox rendering implementation
add for gltf models -> Add helper to each gltf model
basic bbox -> Implement basic bounding box helper
draw lines on 1st camera -> Add test line rendering
create example scene -> Create example helpers scene

**Comment by vorg** [comment]
I would use different color for instanced cubes as they clash with red bounding box
Screenshot 2019-09-10 at 23 01 20

File: index.js [hunk 1]
```diff
@@ -1485,6 +1484,40 @@ Renderer.prototype.draw = function() {
           let cType
           cType = ent.getComponent('Camera')
           if (cType.projection == 'orthographic') {
+            draw = true 
+            console.log(cType)
+            let orthoTransform = ent.getComponent('Transform')
+
+            const dx = (cType.right - cType.left) / (2 / cType.zoom)
```
Comment (simonharrisco) [inline index.js]: In the orthographic camera entity we do a bunch of maths with near/far/zoom and use it to update the orthographic camera matrix. 
https://github.com/pex-gl/pex-renderer/blob/89bc9d3cac2f559c3706435db8941ad4d3f35d4d/camera.js#L183

This maths has to be redone every frame to calculate the view cube to draw helpers, is it worth saving these values as an object property to save on computation/gc of all these declared variables

**Comment by simonharrisco** [comment]
It also seems a little strange that orthographic 'zoom' a higher number means zoomed out. Could just be some bias I have though

**Comment by simonharrisco** [comment]
**Adds helpers for:**
- Point Lights
- Directional Lights
- Spot Lights
- Area Lights
- Axis
- Floor grid
- Perspective Camera
- Orthographic Camera

**Removes**
helpers.js file that was being used in other example files, (replaces functionality)

**Comments / Questions:**
- A review of structure could be nice as it could probably be improved, not sure the best way to do it though
- Some othographic camera questions are above
- At the moment the axis helper is all one color, is there an industry standard for the color of x,y,z? It also shows at world [0,0,0] not the origin of the current entity. Im not sure if this is desired behaviour or not
- Floor grid is currently 1m squares, should this be customisable?
- What is desired behaviour for helper if a light is in the scene as an entity but the light itself is disabled? Currently it shows the helper so long as the entity exists in the scene. 
- Currently the draw command and the vertex buffers are are properties of the renderer object

**Comment by vorg** [comment]
- as long as those are only 3 props (buffers+ cmd) it’s ok
- what about ortho scale instead of zoom? How other engines call it?
- xyz are usually colored rgb :)
- 1m is good default. Whats the size/range?Both size and step should be configurable
- what other engines do for disabled entity helpers? There are 2 options: hide or show at 50% opacity. I would hide.

**Comment by dmnsgn** [comment]
> It also seems a little strange that orthographic 'zoom' a higher number means zoomed out. Could just be some bias I have though

Zoom should be interpreted as size multiplier of camera view planes as we are not using a perspective projection.

Other engines:
- Three: [zoom](https://threejs.org/docs/#api/en/cameras/OrthographicCamera.zoom)
- Unity: [orthoSize](https://docs.unity3d.com/ScriptReference/Camera-orthographicSize.html)
- Babylon: 🤷‍♂️ setting camera.orthoXXX
- Blender: orthographic scale

**Comment by vorg** [comment]
What is the purpose of it though? Why would i set ortho -1,1 and zoom 2 instead of ortho -2,2?

**Comment by dmnsgn** [comment]
For the orbiter I think?

**Pull request merged**